### ISSUE TITLE
Unable to find accurate index due to use of tbody in js

### ISSUE THREAD

**Issue by InsaneSkull** [unclead/yii2-multiple-input#165]
I am working on `divRender` class to  support responsive design and better control over input field size and length.

But this js part not allowing me to find index accurately

```
var getCurrentIndex = function($wrapper) {
     return $wrapper
          .children('.multiple-input-list')
          .children('tbody')
          .children('.multiple-input-list__item')
          .length;
};
```

For now i have to disable that  tbody part as workaround 

```
var getCurrentIndex = function($wrapper) {
     return $wrapper
          .children('.multiple-input-list')
         //.children('tbody')
          .children('.multiple-input-list__item')
          .length;
};
```

But result is broken `tabularInput`. I suggest to introduce class or id selector instead of `tbody`.

**Comment by unclead** [comment]
PR is always welcome 😄

**Comment by InsaneSkull** [comment]
Sure, i will make a PR soon with `DivRender` and  try to fix that js part with callback as you suggested.

**Comment by InsaneSkull** [comment]
@unclead  I don't know js much, can you please suggest, how can i fix that js part?

**Comment by InsaneSkull** [comment]
Basic solution

```
var getCurrentIndex = function($wrapper) {
        var length = $wrapper
            .children('.multiple-input-list')
            .children('tbody')
            .children('.multiple-input-list__item')
            .length;

        if (length < 1) {
            length = $wrapper
                .children('.multiple-input-list')
                .children('.multiple-input-list__item')
                .length;
        }
        return length;
    };
```

**Comment by unclead** [comment]
fixed via 473825099f3f17be13a650ba6d2d901a565b584d
you don't have to use custom callback now because function do not related to table and relies only on widget's css classes

**Comment by InsaneSkull** [comment]
Thanks. I will send PR for `DivRenderer` soon.
